rich/console.py

```diff
@@ -656,8 +656,8 @@ def __init__(
 
         self.is_jupyter = _is_jupyter() if force_jupyter is None else force_jupyter
         if self.is_jupyter:
```

Comment:
For anyone else puzzling over `self.legacy_windows`, which is subtracted from `width` in other places: this isn't needed at all here because `self.legacy_window = 0` when `self.is_jupyter == False`.